.gitignore

```diff
@@ -44,7 +44,6 @@ Thumbs.db
 .tool-versions
 
 # Minder-specific files
-.ssh/
```

Comment:
I'm curious what the motivation is for removing this from the `.gitignore`. Part of our setup generates some secrets which get stored in `.ssh` and we want to prevent folks from accidentally committing them.

Reply:
hey @eleftherias , agree that we need `.ssh` in `.gitignore`, and the reason for removing this one is because there's already another identical one above (line [41](https://github.com/mindersec/minder/pull/5152/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L41))

Reply:
I see it now thanks!